week-2-practise/level-3/6-map.js

```diff
@@ -11,16 +11,18 @@ function tidyUpString(str) {
 }
 
 function captialise(str) {
-  // complete this function
+  str = str.split("");
```

Comment:
It's not good practice to assign a new value to the function parameter - I'd recommend creating a new variable instead.
Well done otherwise! :)